packages/vuetify/src/components/VList/VList.tsx

```diff
@@ -176,6 +183,14 @@ export const VList = genericComponent<new <
     const color = toRef(() => props.color)
     const isSelectable = toRef(() => (props.selectable || props.activatable))
 
+    // Track strategy navigation index
+    const currentNavIndex = useProxiedModel(
```

Comment:
What about naming this "navigationIndex" and instead of wrapping it in a computed, returning it as is on line 383?